docs/public-networks/get-started/connect/sync-node.md

```diff
@@ -4,39 +4,132 @@ description: Full and archive node types
 
 # Sync Besu
 
-Besu supports two node types, commonly referred to as _full nodes_ and _archive nodes_.
+Besu supports two node types, commonly referred to as [full nodes](#run-a-full-node) and
+[archive nodes](#run-an-archive-node).
 
-Full nodes have the current state of the blockchain so cannot serve the network with all data
-requests (for example, the balance of an account at an old block). Full nodes can guarantee the
-latest state for the blockchain (and some older states, but not all). You can check current
-balances, sign and send transactions, and look at current dapp data.
+Full nodes have the current state of the blockchain.
+They can't serve the network with all data requests (for example, the balance of an account at an
+old block).
+Full nodes can guarantee the latest state for the blockchain (and some older states, but not all).
+You can check current balances, sign and send transactions, and look at current dapp data.
 
-Archive nodes have all of this and they also store the intermediary state of every account and
-contract for every block since the genesis block. An archive node can do everything a full node
-does, and it can access historical state data.
+Archive nodes also store the intermediary state of every account and contract for every block since
+the genesis block.
+Archive nodes can do everything full nodes do, and they can access historical state data.
+Archive nodes require more disk space than full nodes.
 
-For Besu on Mainnet, archive nodes [require more disk space](../../concepts/data-storage-formats.md#storage-requirements)
-than full nodes.
+## Sync times
+
+To sync with a public network, Besu runs two processes in parallel: the world state sync and the
+blockchain download.
+
+The following table shows the average world state sync time for each sync mode.
+All times are hardware dependent; this table is based on running AWS instances m6gd.2xlarge.
+Each sync mode also has its own world state database size.
+
+| Sync mode   | Time to sync world state | Disk usage    |
```

Comment:
Can be a second PR, but I think it would be even better to put total initial sync times here since this is what the users will care most about.
Worldstate would be good as well, but maybe as a drill-down.

I am certain that some users don't understand (and even don't care to understand) the details of the sync, they are just looking for an overall time estimate so they can plan and know if it's not going well.

I worry that users will see a number on a nice table without reading the text and assume checkpoint sync only takes 5 hours :)